components/theia/packages/gitpod-extension/src/browser/ports/gitpod-ports-service.ts

```diff
@@ -53,26 +53,22 @@ export class GitpodPortsService {
         return this._ports.values();
     }
 
-    private updatePorts({ added, updated, removed, initial }: DidChangeGitpodPortsEvent): void {
+    private updatePorts({ ports, initial }: DidChangeGitpodPortsEvent): void {
         const toClean = initial ? new Set<number>(this._ports.keys()) : undefined;
```

Comment:
it should happen always now, we can drop `initial`

Reply:
the same in Code: https://github.com/gitpod-io/vscode/commit/875df34b9ecb5059346a2fd46d055fceb7803945#diff-0d515e0b70f335552688d0b80465bc1f0e9525cc759ec8f66371698612447767R163